src/kaleidoscope/hid.cpp

```diff
@@ -51,42 +52,172 @@
 namespace kaleidoscope {
 namespace hid {
 
-void initializeKeyboard() {
-  Keyboard.begin();
-  WITH_BOOTKEYBOARD {
-    BootKeyboard.begin();
-  }
```

Comment:
That's an artifact of how git computes the diff and isn't really related to this actual changes to the file. (And this can change depending on the arguments fed to the diffing algorithm.) I believe this is a non-issue.

Reply:
That's what I expected, but it's still quite surprising. I'm really looking forward to pijul replacing git…